packages/lodestar/src/api/impl/validator/index.ts

```diff
@@ -424,7 +423,6 @@ export function getValidatorApi({chain, config, logger, metrics, network, sync}:
     async publishAggregateAndProofs(signedAggregateAndProofs) {
       notWhileSyncing();
 
-      const seenTimestampSec = Date.now() / 1000;
```

Comment:
By removing the seenTimestamp from here, now the metric is slightly different:
- before: Tracks time attestation is received at the start of handling the REST cal
- now: Track the time the attestation is published to gossip

We should track both, or track the validation time individually. But can be done in a new PR